examples/pytorch/graphsage/train_sampling.py

```diff
@@ -72,8 +72,8 @@ def inference(self, g, x, batch_size, device):
         for l, layer in enumerate(self.layers):
             y = th.zeros(g.number_of_nodes(), self.n_hidden if l != len(self.layers) - 1 else self.n_classes)
 
-            sampler = dgl.sampling.MultiLayerNeighborSampler([None])
-            dataloader = dgl.sampling.NodeDataLoader(
+            sampler = dgl.dataloading.MultiLayerNeighborSampler([None])
```

Comment:
We renaming the sampling package?

Reply:
No.  `sampling` will only contain the operators that will sample for one layer.  Those high-level ones related to dataloading will be put into a new package.

Reply:
I would like to separate the namespace of higher-level data loading utilities from lower-level sampling operators.